src/smart-components/Channel/context/dux/reducers.js

```diff
@@ -274,11 +274,15 @@ export default function reducer(state, action) {
       }
       return {
         ...state,
-        allMessages: state.allMessages.map((m) => (
-          compareIds(m.messageId, action.payload.message.messageId)
-            ? action.payload.message
-            : m
-        )),
+        allMessages: state.allMessages.map((m) => {
+          if (compareIds(m.messageId, message.messageId)) {
+            return message;
+          }
+          if (compareIds(m.parentMessageId, message.messageId)) {
```

Comment:
Probably good to add test(after GA), can you note this part?

Reply:
https://sendbird.atlassian.net/browse/UIKIT-1896